tests/Application/Persistence/RetrySafeEntityManagerTest.php

```diff
@@ -152,6 +152,65 @@ public function testFlushWithRetry(): void
         $this->retrySafeEntityManager->flush();
     }
 
+    public function testRefresh(): void
+    {
+        $underlyingEmProphecy = $this->prophesize(EntityManager::class);
+        $underlyingEmProphecy
+            ->refresh(Argument::type(Donation::class),LockMode::PESSIMISTIC_WRITE)
+            ->shouldBeCalledOnce();
+
+        $retrySafeEntityManagerReflected = new ReflectionClass($this->retrySafeEntityManager);
+
+        $emProperty = $retrySafeEntityManagerReflected->getProperty('entityManager');
+        $emProperty->setValue($this->retrySafeEntityManager, $underlyingEmProphecy->reveal());
+
+        $app = $this->getAppInstance();
+        /** @var Container $container */
+        $container = $app->getContainer();
+        $container->set(EntityManager::class, $underlyingEmProphecy->reveal());
```

Comment:
```suggestion
```

Is there any reason for these lines? The test passes without them.

Reply:
Nope, good catch